src/main/java/com/google/sps/servlets/Statistics.java

```diff
@@ -12,19 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps.servlets.utils;
+package com.google.sps.servlets;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class Statistics {
   // Aggregated score for a list of comments  
-  // Contains sentiment scores in the range [-1, 1] with 0.2 intervals.
-  private List<Double> aggregateValues;
+  // Contains number of sentiment scores in the range [-1, 1] with 0.2 intervals.
+  private List<Double> sentimentScores;
+  private Map<Range, Integer> aggregateValues;
   private double averageScore;
 
-  public Statistics(List<Double> aggregateValues, double averageScore) {
-    this.aggregateValues = aggregateValues;
-    this.averageScore = averageScore;
+  public Statistics(List<Double> sentimentScores) {
+    this.sentimentScores = sentimentScores;
+    getAggregateScores();
```

Comment:
In line with my above comment, maybe we should change both of these methods to take a `List<Double> sentimentScores` argument. 

In addition, to match their intent, they should likely be named to `setAggregateScores` and `setAverageScore`.